.github/labeler.yml

```diff
@@ -21,6 +21,11 @@
   - "**/*.dts"
   - "include/devicetree/*"
   - "include/dt-bindings/**/*"
+  - "scripts/dts/**/*"
```

Comment:
Isn't it duplicated with "Device Tree Tooling" below?

Reply:
Want to have the 'Area: Device Tree' label as a catch all.

Reply:
Should we have `Area: Device Tree API` for `include/*`?

Reply:
>  Should we have Area: Device Tree API for include/*?

I think that should be `include/devicetree.h`, right?

Reply:
I meant `Area: Devicetree API` for the files under `include/` that are relevant to devicetree:

    - "include/devicetree/*"
    - "include/devicetree.h"

not everything under `include/`.  (I agree about the spelling w/o the space.)